src/i18n/en_GB.ts

```diff
@@ -496,7 +496,7 @@ export default {
             maxPersonnel: 1,
             schooling: 'Rescue - Ambulance Officer',
             shownSchooling: 'Ambulance Officer',
-            special: 'Required once you have built 15 Rescue stations',
+            special: 'Required once you have built 15 Rescue stations. It is required for missions that can spawn with over 20 paitents to help command the scene it is a dual unit betweeen a OTL and a Ambulance Officer Requirements',
```

Comment:
Same with patients 
betweeen => between
an OTL and an Ambulance

Reply:
Is it needed to say that it is an dual unit? That info is not at the other dual units

Reply:
It is I have done it for other dual units and it is not a ambulance

Reply:
I don't mean it's acting as ambulance. But you wrote `a` ambulance, and `a` OTL. It should be `an` in both cases